packages/one-app-bundler/utils/loadExternalsPackageJson.js

```diff
@@ -1,21 +0,0 @@
-/*
- * Copyright 2023 American Express Travel Related Services Company, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- */
-
-// Because this file uses very modern node features, it may not be parsed by every IDE.
-// As long as eslint and jest are happy when they are run by yarn, all is ok.
-const loadExternalsPackageJson = async (externalName) => (await import(`${externalName}/package.json`, { assert: { type: 'json' } })).default;
-
-export default loadExternalsPackageJson;
```

Comment:
why has this utility been removed? looks like we use it at least in two places, might be worth keeping it and refactor it

Reply:
It was only created because of the the need to dynamically import, as part of this PR.

Jonny has recently changed the way we load externals package json's, so the hack is no-longer needed.

in the full change set, this util file is 'never created', since it was both added and removed for this PR